README.Rmd

```diff
@@ -25,7 +25,7 @@ knitr::opts_chunk$set(
 btw helps you describe your computational environment to LLMs.
 
 * When used **interactively**, `btw()` assembles context on your R environment, package documentation, and working directory, copying the results to your clipboard for easy pasting into chat interfaces.
-* The `btw()` function wraps methods that can be easily incorporated into **ellmer tool calls** for describing various kinds of objects in R. To equip your ellmer chat with the ability to peruse documentation, check out the objects in your R environment, and explore your working directory, pass your chat to `btw_register_tools()`.
+* The `btw()` function wraps methods that can be easily incorporated into **ellmer tool calls** for describing various kinds of objects in R. To equip your ellmer chat with the ability to peruse documentation, check out the objects in your R environment, and explore your working directory, call `chat$set_tools(btw_tools())`.
```

Comment:
I feel bad mentioning `$set_tools()` here without acknowledging that it will fully set the btw tools.

That said, I think we should merge this as-is but before CRAN release we should give the README a pass. We can probably streamline it considerably now that we have more documentation in other places, and we may want to adjust which functions we're highlighting up-front. E.g. here we might prefer to highlight `btw_client()` and _later_ discuss that the tools behind `btw()` and `btw_client()` are available in `btw_tools()` if you want to forge your own path

Reply:
Heard. 09c3a3a is related but doesn't fully resolve.